Launcher/app/reducers/fileLoader.js

```diff
@@ -114,11 +126,29 @@ function changeFolderSelection(state, action) {
       game: game,
     };
   });
+  const numberOfPages = Math.ceil(files.length / 10);
+  const paginatedFiles = [];
+  for (let pagesX = 0; pagesX <= numberOfPages; pagesX++) {
```

Comment:
This loop could just be `_.chunk` I think?

Reply:
I've never used chunk before. Is that lodash?